File: android/src/main/java/ch/nevis/mobile/authentication/sdk/react/model/message/outgoing/ChannelOutMessage.kt
/*
 * Copyright © 2023 Nevis Security AG. All rights reserved.
 */

package ch.nevis.mobile.authentication.sdk.react.model.message.outgoing

import kotlinx.serialization.Serializable

@Serializable
sealed class ChannelOutMessage {
    abstract val operationId: String
}

File: ios/Model/Message/Outgoing/ChannelOutMessage.swift
//
// Copyright © 2023 Nevis Security AG. All rights reserved.
//

import Foundation

class ChannelOutMessage: Encodable {
	// MARK: Properties

	let operationId: String

	// MARK: Initialization

	init(operationId: String) {
		self.operationId = operationId
	}
}
